helm/opendistro-es/templates/elasticsearch/es-config-secret.yaml

```diff
@@ -28,4 +28,22 @@ data:
   {{- if .Values.elasticsearch.log4jConfig }}
   log4j2.properties: {{ .Values.elasticsearch.log4jConfig | b64enc | quote }}
   {{- end }}
-  logging.yml: {{ toYaml .Values.elasticsearch.loggingConfig | b64enc | quote }}
+  logging.yml:  {{ toYaml .Values.elasticsearch.loggingConfig | b64enc | quote }}
+
+---
+{{- if .Values.elasticsearch.securityConfig.enabled }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ template "opendistro-es.fullname" . }}-es-config-secrets
```

Comment:
Where are we mounting this secret?

Reply:
This is for the security config secrets (config.yml, internal_users.yml, roles.yml etc) so that we can easily provide these configs as:
config: 
      # config.yml: |-
      # internal_users.yml: |-
      # roles.yml: |-
      # rolesMapping.yml: |-
      # tenants.yml: |-